CI: Windows wheel builds failing during vcpkg pull

All Windows jobs in the Build & Publish workflow are failing in the cibuildwheel before_all step. This affects every open PR.

### Error
```bash
cibuildwheel: Command git -C C:/vcpkg pull && vcpkg install --triplet x64-windows failed with code 1.
Error: Process completed with exit code 1.
```

### Root cause

A refreshed windows-latest runner image broke git -C C:/vcpkg pull — the bundled C:/vcpkg is in a detached-HEAD state, so git pull exits non-zero and && fails the whole command. (macOS runners are also emitting new brew tap-trust warnings, confirming a broader image refresh.)

### Proposed fix
Remove the git -C C:/vcpkg pull and just run vcpkg install --triplet x64-windows.

--- a/.github/actions/setup_xtgeo/action.yml
+++ b/.github/actions/setup_xtgeo/action.yml
@@ -24,7 +24,6 @@ runs:
       if: ${{ inputs.os == 'windows-latest' }}
       shell: bash
       run: |
-        git -C C:/vcpkg pull
         vcpkg install --triplet x64-windows
 
     - name: Install macOS dependencies

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -137,10 +137,7 @@ test-command = [
 ]
 
 [tool.cibuildwheel.windows]
-before-all = [
-    "git -C C:/vcpkg pull",
-    "vcpkg install --triplet x64-windows"
-]
+before-all = "vcpkg install --triplet x64-windows"
 
 [tool.cibuildwheel.linux]
 before-all = "yum update -y && yum install -y eigen3-devel fmt-devel || (apt-get update && apt-get install -y libeigen3-dev libfmt-dev)"

